apps/studio/components/interfaces/SQLEditor/SQLTemplates/SQLQuickstarts.tsx

```diff
@@ -52,35 +54,48 @@ const SQLQuickstarts = () => {
       })
       snapV2.addSnippet({ projectRef: ref, snippet })
       snapV2.addNeedsSaving(snippet.id)
+
+      const store = getTabsStore(ref)
+      const tabId = `sql-${snippet.id}`
+      store.openTabs = [...store.openTabs, tabId]
+      store.tabsMap[tabId] = {
+        id: tabId,
+        type: 'sql',
+        label: name,
+        metadata: { sqlId: snippet.id, name },
+      }
+      store.activeTab = tabId
```

Comment:
Can this be converted into a method in the store? Seems kinda scary this logic is just hanging out here 😅

Reply:
don't even need this. removing